application.go

```diff
@@ -62,6 +62,7 @@ type Application struct {
 	Constraints                *[][]string          `json:"constraints,omitempty"`
 	Container                  *Container           `json:"container,omitempty"`
 	CPUs                       float64              `json:"cpus,omitempty"`
+	GPUs                       float64              `json:"gpus,omitempty"`
```

Comment:
According to the documentation, the default value for the `gpu` property is 0.

If we want users be able to reset the number of GPUs to zero, the field must be a pointer (`*float64`).